configs/ozone_config/frontend/ozone-frontend-config.json

```diff
@@ -138,6 +138,7 @@
         "District": "District",
         "contactSection": "Contact Details",
         "demographicsSection": "Basic Info",
+        "additionalInformationSection": "Additional Information",
```

Comment:
@vasharma05 @kazlaw following the changes done in [this](https://github.com/openmrs/ozone-distro-cambodia/pull/246/files#diff-8f48a4ca8c8ba2aa336358396f7ececb07a4a1b373581ebd122fc4fb0ed99027R4-R5) section, won't these strings require a change?

Reply:
@kdaud please see [here](https://mekomsolutions.slack.com/archives/G421UNF5L/p1698747057532279?thread_ts=1698734503.031209&cid=G421UNF5L)